contracts/mechs/nevermined/BalanceTrackerNvmSubscriptionNative.sol

```diff
@@ -38,13 +32,13 @@ error Overflow(uint256 provided, uint256 max);
 /// @param amount Value amount.
 error NoDepositAllowed(uint256 amount);
 
-contract BalanceTrackerNvmSubscription is BalanceTrackerBase {
-    event SubscriptionSet(address indexed token,  uint256 indexed tokenId);
-    event WithdrawSubscription(address indexed account, address indexed token, uint256 indexed tokenId, uint256 amount);
```

Comment:
Not needed, subscription-based withdraws work regular token way, since the balance for subscription in tokens is already on the contract.